cumulusci/cumulusci.yml

```
@@ -1187,6 +1190,13 @@ services:
         description: The email address to used by Github tasks when an operation requires an email address.
         required: True
     validator: cumulusci.core.github.validate_service
+  marketing-cloud:
```

```suggestion
  marketing_cloud:
```

to follow the pattern established by `connected_app`